.github/workflows/build-meson.yml

```diff
@@ -0,0 +1,47 @@
+name: build-meson
+on: [push, pull_request]
+
+jobs:
+  buildmeson:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          submodules: "recursive"
+          fetch-depth: 0
+      - name: Install Conda environment
+        uses: mamba-org/setup-micromamba@v1
+        with:
+          environment-name: highsdev
+          create-args: >-
+            python==3.8
+            meson
+            pkgconfig
+            openblas
+            ninja
+            zlib
+            gfortran
+          cache-environment: true
+          init-shell: >-
+            bash
+            zsh
+      - name: Build
+        shell: bash -l {0}
+        run: |
+          meson setup bbdir -Duse_zlib=True --prefix $CONDA_PREFIX
+          meson install -C bbdir
+      - name: Build Fast
+        shell: bash -l {0}
+        run: |
+          meson setup bbdir_fast -Duse_zlib=True -Dfast_build=True --prefix $CONDA_PREFIX
+          meson install -C bbdir_fast
+      - name: Build Static
+        shell: bash -l {0}
+        run: |
+          meson setup bbdir_static -Duse_zlib=True --default-library static --prefix $CONDA_PREFIX
+          meson install -C bbdir_static
+      - name: Build Fortran
+        shell: bash -l {0}
+        run: |
+          meson setup bbdir_fortran -Duse_zlib=True -Dwith_fortran=True
+          meson install -C bbdir_fortran
```

Comment:
These don't all need to be tested here in CI like this. Probably just a single one.